styles.css

```diff
@@ -4,7 +4,7 @@ body {
   font-family: 'Raleway', sans-serif;
 }
 
-.jamazonText {
+.headerText {
```

Comment:
Stick with `kebab-case` convention for your CSS class names and ids.

Reply:
![screen shot 2018-07-05 at 1 12 38 pm](https://user-images.githubusercontent.com/13921070/42345721-1d328c64-8055-11e8-871f-92cce95e589c.png)